src/components/Navbar.js

```diff
@@ -1,34 +1,35 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
 import { AiOutlineBars } from "react-icons/ai";
 import { RiCloseLine } from "react-icons/ri";
 import { BsCode, BsCodeSlash} from "react-icons/bs";
 import Button2 from "./Button2";
 import "./Button2.css";
 import "./Navbar.css";
 import { useNavigate } from "react-router-dom";
+import {UserContext} from '../util/context';
+import { Button} from 'react-bootstrap';
 
 const Navbar = () => {
   const [showMenu, setShowMenu] = useState(false);
-  // const toast = useToast();
   let navigate = useNavigate(); 
+  const {setAuthorization} = useContext(UserContext);
 
   const toggleMenu = () => {
     setShowMenu(!showMenu);
   };
 
-  /*const handleMenuChange = (e) => {
+  const handleClick = async (e) => {
+    e.preventDefault();
     if (e.target.value === 'logout') {
-        window.localStorage.clear();
-        toast({
-            title: 'Logged out successfully.',
-            status: 'success',
-            duration: 3000,
-            isClosable: true,
-        });
-        navigate('/auth/signin');
-    } 
-};
-*/
+      setAuthorization('');
+      window.localStorage.clear();
```

Comment:
This should be done in the wrapper. You can make a function `logout` that you pass through the context.